### PR TITLE
Update base ubi image to lalest ubi micro 9.3-6

### PR DESCRIPTION
**Pull request by alikdell**
# Description
Update base ubi image to lalest ubi micro 9.3-6. Update scripts to use common CSM ubi base image location.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| #https://github.com/dell/csm/issues/1031 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Run cert-csi suit for unity

[2023-11-07 13:55:04]  INFO Avg time of a run:	 73.61s
[2023-11-07 13:55:04]  INFO Avg time of a del:	 18.09s
[2023-11-07 13:55:04]  INFO Avg time of all:	 95.35s
[2023-11-07 13:55:04]  INFO During this run 100.0% of suites succeeded
CMD RETURNCODE IS: 0
ALL CERT-CSI RETURNCODEs: 0,0,0
All test(s) passed

### CODE REVIEW

**Review by donatwork — DISMISSED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -28,9 +28,10 @@ function git_version {
    echo Target Version=$VERSION
 }
 
+
 function build_image {
-   # Tag corresponding to digest sha256:630cf7bdef807f048cadfe7180d6c27eb3aaa99323ffc3628811da230ed3322a for ubi9 micro is 9.2-13
-   bash build_ubi_micro.sh registry.access.redhat.com/ubi9/ubi-micro@sha256:630cf7bdef807f048cadfe7180d6c27eb3aaa99323ffc3628811da230ed3322a
+   # Tag corresponding to digest sha256:b7d8fd2840e92e8adc68414e13d859763ef33c3d4c4e27f910e939c00d642c29 for ubi9 micro is 9.3-6
```
Comment (tdawe) [inline build.sh:33]: The common base image can be accessed by downloading the https://raw.githubusercontent.com/dell/csm/main/config/csm-common.mk file and using the value of DEFAULT_BASEIMAGE

Comment (tdawe) [reply build.sh:33]: Example in the Makefile for karavi-metrics-powerflex: https://github.com/dell/karavi-metrics-powerflex/pull/158/files

Comment (AronAtDell) [reply build.sh:33]: @alikdell , @tdawe - Let's ensure that we drive a consistent approach all repositories that build images to use the same mechanism that you described. Thanks for your attention to this.

**Review by AronAtDell — APPROVED** [review]
Thanks @alikdell! Looks good. Please influence the changes to follow this pattern in the other repositories to drive a single, consistent approach to defining our base image.

**Review by gallacher — APPROVED** [review]

**Review by Deepak-Ghivari — APPROVED** [review]

**Review by panigs7 — APPROVED** [review]

**Pull request merged**